### PR TITLE
Master bug 566467

### PR DESCRIPTION
**Pull request by aliakar82**

### CODE REVIEW

**Review by pdulth — CHANGES_REQUESTED** [review]
can you use formatOutput="true" in your build.xml to generate formatted htmls.
Thanks

https://github.com/eclipse/capella/blob/master/doc/plugins/org.polarsys.capella.ui.doc/build.xml#L38

**Review by pdulth — APPROVED** [review]

**Review by pdulth — DISMISSED** [review]

**Review by pdulth — APPROVED** [review]

File: doc/plugins/org.polarsys.capella.th.doc/html/Interfaces and Data Modeling/Advanced Constructs.html [hunk 1]
```diff
@@ -197,7 +198,7 @@ <h2 id="Union_.2F_Variant">Union / Variant</h2>
 		<p>Differences between Union and Variant are the following:</p>
 		<ul>
 			<li>In the Union, one value of the discriminant must be associated to at most one property and a property must always be associated to a value of the discriminant;</li>
-			<li>In the Variant, one value of the discriminant can be associated to any number (0 or more) of properties. By convention a variant property that has no discriminant value associated is deemed to be associated to all values ​​of the discriminant: it is the fixed part.</li>
+			<li>In the Variant, one value of the discriminant can be associated to any number (0 or more) of properties. By convention a variant property that has no discriminant value associated is deemed to be associated to all values ??of the discriminant: it is the fixed part.</li>
```
Comment (sandupostaru) [inline doc/plugins/org.polarsys.capella.th.doc/html/Interfaces and Data Modeling/Advanced Constructs.html:201]: `??` typo?

File: doc/plugins/org.polarsys.capella.th.doc/html/Interfaces and Data Modeling/Interfaces and Data Modeling.html [hunk 2]
```diff
@@ -0,0 +1,1448 @@
+<?xml version='1.0' encoding='utf-8' ?><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```
Comment (sandupostaru) [inline doc/plugins/org.polarsys.capella.th.doc/html/Interfaces and Data Modeling/Interfaces and Data Modeling.html:1]: Was this file not generated ?

Comment (pdulth) [reply doc/plugins/org.polarsys.capella.th.doc/html/Interfaces and Data Modeling/Interfaces and Data Modeling.html:1]: idk

File: doc/plugins/org.polarsys.capella.th.doc/html/Interfaces and Data Modeling/Advanced Constructs.html (hunk 1 above)
Comment (pdulth) [reply doc/plugins/org.polarsys.capella.th.doc/html/Interfaces and Data Modeling/Advanced Constructs.html:201]: indeed

**Review by sandupostaru — APPROVED** [review]

**Pull request merged**